projects/ui-policy-studio-angular/src/lib/components/flow-form-dialog/flow-mcp-form-dialog/gio-ps-flow-mcp-form-dialog.component.ts

```diff
@@ -0,0 +1,203 @@
+import { MatInputModule } from '@angular/material/input';
+import { MatSlideToggleModule } from '@angular/material/slide-toggle';
+import {
+  GioBannerModule,
+  GioElAssistantComponent,
+  GioFormSlideToggleModule,
+  GioFormTagsInputModule,
+  GioIconsModule,
+} from '@gravitee/ui-particles-angular';
+
+import { GioPolicyStudioFlowFormDialogResult } from '../gio-ps-flow-form-dialog-result.model';
+import { FlowVM } from '../../../policy-studio/gio-policy-studio.model';
+import { ConditionSelector, McpSelector } from '../../../models';
+
+export type GioPolicyStudioFlowMcpFormDialogData = {
+  flow?: FlowVM;
+  parentGroupName?: string;
+};
+
+const MCP_METHODS: {
+  groupLabel: string;
+  groupOptions: { value: string; label: string }[];
+}[] = [
+  {
+    groupLabel: 'Lifecycle Methods',
+    groupOptions: [
+      { value: 'initialize', label: 'initialize' },
+      { value: 'notifications/initialized', label: 'notifications/initialized' },
+      { value: 'ping', label: 'ping' },
+      { value: 'notifications/progress', label: 'notifications/progress' },
+    ],
+  },
+  {
+    groupLabel: 'Tool Methods',
+    groupOptions: [
+      { value: 'tools/list', label: 'tools/list' },
+      { value: 'tools/call', label: 'tools/call' },
+      { value: 'notifications/tools/list_changed', label: 'notifications/tools/list_changed' },
+    ],
+  },
+  {
+    groupLabel: 'Resources Methods',
+    groupOptions: [
+      { value: 'resources/list', label: 'resources/list' },
+      { value: 'resources/read', label: 'resources/read' },
+      { value: 'notifications/resources/list_changed', label: 'notifications/resources/list_changed' },
+      { value: 'notifications/resources/updated', label: 'notifications/resources/updated' },
+      { value: 'resources/templates/list', label: 'resources/templates/list' },
+      { value: 'resources/subscribe', label: 'resources/subscribe' },
+      { value: 'resources/unsubscribe', label: 'resources/unsubscribe' },
+    ],
+  },
+  {
+    groupLabel: 'Prompt Methods',
+    groupOptions: [
+      { value: 'prompts/list', label: 'prompts/list' },
+      { value: 'prompts/get', label: 'prompts/get' },
+      { value: 'notifications/prompts/list_changed', label: 'notifications/prompts/list_changed' },
+      { value: 'completion/complete', label: 'completion/complete' },
+    ],
+  },
+  {
+    groupLabel: 'Logging Methods',
+    groupOptions: [
+      { value: 'logging/setLevel', label: 'logging/setLevel' },
+      { value: 'notifications/message', label: 'notifications/message' },
+    ],
+  },
+  {
+    groupLabel: 'Roots Methods',
+    groupOptions: [
+      { value: 'roots/list', label: 'roots/list' },
+      { value: 'notifications/roots/list_changed', label: 'notifications/roots/list_changed' },
+    ],
+  },
+  {
+    groupLabel: 'Sampling Methods',
+    groupOptions: [{ value: 'sampling/createMessage', label: 'sampling/createMessage' }],
+  },
+  {
+    groupLabel: 'Elicitation Methods',
```

Comment:
I didn't know that word `Elicitation` 🙈

Reply:
haha me too. I took the list directly from the Spec 
https://modelcontextprotocol.io/docs/learn/client-concepts#elicitation